source/documentation/administration_guide/topics/Configuring_the_Red_Hat_Virtualization_Manager_to_Send_SNMP_Traps.adoc

```diff
@@ -1,46 +1,107 @@
 
+* An SNMP username and password.
+
 
 [NOTE]
 ====
-The {virt-product-fullname} {engine-name} provides management information bases at */usr/share/doc/ovirt-engine/mibs/OVIRT-MIB.txt* and */usr/share/doc/ovirt-engine/mibs/REDHAT-MIB.txt*. Load the MIBs in your SNMP manager before proceeding.
+The {virt-product-fullname} {engine-name} provides management information bases at `/usr/share/doc/ovirt-engine/mibs/OVIRT-MIB.txt` and `/usr/share/doc/ovirt-engine/mibs/REDHAT-MIB.txt`. Load the MIBs in your SNMP manager before proceeding.
 ====
-Default SNMP configuration values exist on the {engine-name} in the events notification daemon configuration file */usr/share/ovirt-engine/services/ovirt-engine-notifier/ovirt-engine-notifier.conf*. The values outlined in the following procedure are based on the default or example values provided in that file. It is recommended that you define an override file, rather than edit the *ovirt-engine-notifier.conf* file, to persist your configuration options after changes such as system upgrades.
 
+Default SNMP configuration values exist on the {engine-name} in the events notification daemon configuration file `/usr/share/ovirt-engine/services/ovirt-engine-notifier/ovirt-engine-notifier.conf`. The values outlined in the following procedure are based on the default or example values provided in this file. Do not edit this file directly, because system changes, such as upgrades, might remove any changes you make to this file. Instead, copy this file to `/etc/ovirt-engine/notifier/notifier.conf.d/<__integer__>-snmp.conf`, where `<__integer__>` is a number indicating the priority with which the file should run.
 
-*Configuring SNMP Traps on the {engine-name}*
+.Procedure
 
-. On the {engine-name}, create the SNMP configuration file:
+. On the {engine-name}, create the SNMP configuration file with the file name `<__integer__>-snmp.conf`, where `<__integer__>` is an integer that indicates the order in which files are processed. For example:
 +
 [options="nowrap" subs="normal"]
 ----
 # vi /etc/ovirt-engine/notifier/notifier.conf.d/20-snmp.conf
 ----
 +
-. Specify the SNMP manager(s), the SNMP community, and the OID in the following format:
+[TIP]
+====
+Copy the default SNMP settings from the events notification daemon configuration file `/usr/share/ovirt-engine/services/ovirt-engine-notifier/ovirt-engine-notifier.conf`. This file includes inline comments for all settings.
+====
+
+. Specify the SNMP manager(s), the SNMP community, and the OID in the format in this example:
 +
 [options="nowrap" subs="normal"]
 ----
-SNMP_MANAGERS="_manager1.example.com_ _manager2.example.com:162"_
+SNMP_MANAGERS="_manager1.example.com_ _manager2.example.com:162_"
 SNMP_COMMUNITY=public
```

Comment:
Not relevant on v3

Reply:
@santos1709 What's not relevant on v3? the entire SNMP_MANAGERS line?

Reply:
the community, selected wrong line